src/Files.App/App.xaml.cs

```diff
@@ -97,6 +100,9 @@ async Task ActivateAsync()
 				var userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
 				var isLeaveAppRunning = userSettingsService.GeneralSettingsService.LeaveAppRunning;
 
+				Ioc.Default.GetRequiredService<IRealTimeLayoutService>().UpdateCulture(new(AppLanguageHelper.PreferredLanguage.Code));
+				MainWindow.Instance.InitializeContentLayout();
```

Comment:
Shouldn't this be called from MainWindow's constructor after InitializeComponent call?

Reply:
For calling, the culture must be already set by realTimeLayoutService.UpdateCulture

Reply:
I mean InitializeContentLayout shouldn't be called in MainWindow's constructor as others?

Reply:
Yes I understand, the generator works with the IoC as part of the initialization to implement the service. But the MainWindow constructor is called before the IoC is implemented.